pkg/deployments/tasks/20220704-timelock-authorizer/input/mainnet.ts

```diff
@@ -0,0 +1,250 @@
+import { DAY } from '@balancer-labs/v2-helpers/src/time';
+import Task, { TaskMode } from '../../../src/task';
+import { flatten } from 'lodash';
+import { DelayData, RoleData } from './types';
+import { ANY_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+
+const EVERYWHERE = ANY_ADDRESS;
+
+const Vault = new Task('20210418-vault', TaskMode.READ_ONLY, 'mainnet');
+const ProtocolFeesCollector = new Task('20210418-vault', TaskMode.READ_ONLY, 'mainnet');
+const ProtocolFeesWithdrawer = new Task('20220517-protocol-fee-withdrawer', TaskMode.READ_ONLY, 'mainnet');
+
+const BalancerTokenAdmin = new Task('20220325-balancer-token-admin', TaskMode.READ_ONLY, 'mainnet');
+const BalancerMinter = new Task('20220325-gauge-controller', TaskMode.READ_ONLY, 'mainnet');
+const GaugeAdder = new Task('20220628-gauge-adder-v2', TaskMode.READ_ONLY, 'mainnet');
+const GaugeController = new Task('20220325-gauge-controller', TaskMode.READ_ONLY, 'mainnet');
+const VotingEscrowDelegationProxy = new Task('20220325-ve-delegation', TaskMode.READ_ONLY, 'mainnet');
+const SmartWalletChecker = new Task('20220420-smart-wallet-checker', TaskMode.READ_ONLY, 'mainnet');
+const LiquidityGaugeV5 = new Task('20220325-mainnet-gauge-factory', TaskMode.READ_ONLY, 'mainnet');
+
+const BalancerRelayer = new Task('20211203-batch-relayer', TaskMode.READ_ONLY, 'mainnet');
+const LidoRelayer = new Task('20210812-lido-relayer', TaskMode.READ_ONLY, 'mainnet');
+// https://forum.balancer.fi/t/proposal-balancer-v2-authorize-gnosis-protocol-v2-contracts-as-a-vault-relayer/1938
+// https://etherscan.io/address/0xc92e8bdf79f0507f65a392b0ab4667716bfe0110#code
+const GnosisProtocolRelayer = '0xc92e8bdf79f0507f65a392b0ab4667716bfe0110';
+
+const SingleRecipientGauge = new Task('20220325-single-recipient-gauge-factory', TaskMode.READ_ONLY, 'mainnet');
+
+const StablePool = new Task('20210624-stable-pool', TaskMode.READ_ONLY, 'mainnet');
+const MetaStablePool = new Task('20210727-meta-stable-pool', TaskMode.READ_ONLY, 'mainnet');
+const StablePhantomPool = new Task('20211208-stable-phantom-pool', TaskMode.READ_ONLY, 'mainnet');
+const WeightedPool = new Task('20210418-weighted-pool', TaskMode.READ_ONLY, 'mainnet');
+const AaveLinearPool = new Task('20211208-aave-linear-pool', TaskMode.READ_ONLY, 'mainnet');
```

Comment:
Aren't we missing the lbp factory?

Reply:
This permission wasn't previously granted and I restricted this PR to grant only existing permissions.